R/Transformations.R

```diff
@@ -392,7 +392,7 @@ HarmonizeData <- function(.data, ref, transform = TRUE, flag = TRUE) {
   ) %in% colnames(.data)) == TRUE) {
 
     # if class(ResultMeasureValue) != numeric, run special char function
-    if (class(.data$ResultMeasureValue) != "numeric") {
+    if (!is.numeric(.data$ResultMeasureValue) ) {
```

Comment:
These are relatively new NOTES in R. I think they started in ~4.1 or so. Anyway, it's always good to remove as many notes as possible.

Reply:
Thank you so much for your help with these notes! It's been on my list to address, but I wasn't sure how to do so.